sklearn/feature_extraction/text.py

```diff
@@ -876,14 +876,17 @@ class CountVectorizer(_VectorizerMixin, BaseEstimator):
 
     ngram_range : tuple (min_n, max_n), default=(1, 1)
         The lower and upper boundary of the range of n-values for different
+        word n-grams or char n-grams to be extracted. All values of n such
```

Comment:
It seems that you still have twice "n-grams to be extracted. All values of n such that min_n <= n <= max_n will be used." in this paragraph. Did you push all your changes ?

Reply:
I added n-grams after 'word' because, without it, it is not trivial for end-user that word n-grams is intended. Do you disagree?

Reply:
Now I see your point. You are right. Will fix!